tests/AbstractBaseApiTestCase.php

```diff
@@ -70,6 +74,7 @@ protected function getAuthenticatedClient(string $role = 'ROLE_EDITOR'): Client
 
             $user->addUserRoleTenant($userRoleTenant);
 
+            $manager->persist($userRoleTenant);
```

Comment:
Should not be necessary because `User` has `cascade: ['persist', ...` on the relation
```
    #[ORM\OneToMany(targetEntity: UserRoleTenant::class, mappedBy: 'user', cascade: ['persist', 'remove'], orphanRemoval: true)]
    private Collection $userRoleTenants;
```

Reply:
Fixed